snippets/project-tree.ttl

```diff
@@ -0,0 +1,69 @@
+PREFIX st: <http://www.w3.org/ns/shapetrees#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX ex: <http://www.example.com/ns/ex#>
+
+<> st:describedBy ex:project-en, ex:project-es, ex:project-nl, ex:project-ko .
```

Comment:
Would it be possible to have a direct link between the shape tree and its description? Here it seems that we have a link between a document that includes multiple shape tree definitions to descriptions which feels like weird indirection.

Reply:
Note that the shape tree description links directly to the shape tree via `st:describes`. The benefit of having the descriptions link to the shape vs. the other way around, is that we can expand those more freely without having to directly modify the composition of the shape trees definitions themselves.

Reply:
Note issue created to track this outside this PR: #72